libs/dyn/convert/normalize.go

```diff
@@ -98,8 +98,9 @@ func (n normalizeOptions) normalizeStruct(typ reflect.Type, src dyn.Value, seen
 					diags = diags.Append(diag.Diagnostic{
 						Severity: diag.Warning,
 						Summary:  fmt.Sprintf("unknown field: %s", pk.MustString()),
-						Location: pk.Location(),
-						Path:     path,
+						// Show all locations the unknown field is defined at.
```

Comment:
You could omit the comment. We normalize after each load so it can only be defined in one spot. If we see the same path be used multiple times across multiple files, it will lead to multiple warnings (this is fine).

Reply:
Makes sense. Removed the comment.